composer.json

```
@@ -53,7 +53,7 @@
     "require": {
         "php": ">=7.4 <8.3",
         "atk4/data": "dev-develop",
-        "nyholm/psr7": "^1.4",
+        "nyholm/psr7": "^1.6.0",
```

```suggestion
        "nyholm/psr7": "^1.6",
```

looks fine - https://github.com/Nyholm/psr7/blob/1.6.0/CHANGELOG.md

make sure you change it also in the `require-release`!